core/frontend/src/tools/ToolAdmin.ts

```diff
@@ -1008,15 +1003,31 @@ export class ToolAdmin {
         if (undefined !== tool && isValidLocation)
           tool.receivedDownEvent = true;
 
-        return this.onStartDrag(ev, isValidLocation ? tool : undefined);
+        await this.onStartDrag(ev, isValidLocation ? tool : undefined);
+        return;
```

Comment:
superfluous `return`.

Reply:
Keith changed that...he preferred not return EventHandled when no one checks the status...

Reply:
no it's not.

Reply:
I see -  you changed it after I read it.

Reply:
haha - you were commenting about the `return` that i removed, but your comment happened to fall on another `return` after my changes.